training.py

```diff
@@ -261,22 +267,21 @@ def predict(self, batch_data, threshold=0.25):
         Make predictions with a 30 second interval.
         
         Inputs:
-        batch_data: torch.util.data.Dataset. Should be dataloader.PreprocessedParsedReplayDataset class.
-        Outputs:
-        predictions: torch.Tensor, size=(batch_size, L). Predictions of the probability that radiant will win. batch_size should only be 1. L is the length of the game.
+            batch_data: torch.util.data.Dataset. Should be dataloader.PreprocessedParsedReplayDataset class.
+        Returns:
+            predictions: torch.Tensor, size=(batch_size, L). Predictions of the probability that radiant will win. batch_size should only be 1. L is the length of the game.
         """
         with torch.no_grad():
             dim = batch_data["features"].shape
             
             #If batch_size != 1
             if len(dim) == 3:
-                self.model.hidden_cell = self.model.init_hidden()
+                self.model.hidden_cell = self.model.init_hidden(self.model.batch_size)
```

Comment:
Overall, file changes look good, mostly changes to document formatting.

Noticed this at the end though, you have this repeated pattern `self.model.init_hidden(self.model.batch_size)` throughout your code. When that happens, one thing to consider is writing for your class:

```
def init_hidden():
    self.model.hidden_cell = self.model.init_hidden(self.model.batch_size)
```

One of the motivations for this is that: you're essentially writing a wrapper class for a pytorch neural network. It is helpful sometimes to isolate a lot of the repeated interactions with the pytorch package in a single function. Since it's just one thing right now, I'll say not a necessary change. But, if you see anything else that's a repeated interaction that needs to be initialized for every run, see about consolidating all of them into a single function.

Reply:
Yeah, I was using that. But then I realized, since I need to pass in a single input to generate the graph for a single game, I have to change the `self.model.batch_size` to do that. Then later on, if I want to do something with the model, I need to change the `self.model.batch_size` back. This is the reason I decided to keep it this way, using a argument for the `init_hidden` function. Probably there is a better solution though.